BackEnd/src/main/java/com/exadel/carpoolfree/model/Drive.java

```diff
@@ -4,18 +4,7 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
-import java.time.LocalDateTime;
+import javax.persistence.*;

```

Comment:
Please avoid using asterisk imports, you can set it up in Settings > Code Style > Java > imports

Reply:
done